### PR TITLE
Adding workflow for isolated unit tests

### PR DESCRIPTION
**Pull request by BT-rmartin**

### CODE REVIEW

**Review by BT-fgarbely — APPROVED** [review]
LGTM from my side (as discussed in the meet)

**Comment by BT-rmartin** [comment]
@BT-fgarbely Added now the logic in Python!! Was quite challenging but I got it, so "easier" to review. 
Added the logic discussed this morning: 
- By default it just focuses on modules of the project repo
- If added --test-submodules it will take into account modules from bt repos and those added as parameter for --test-submodules
- In any case, if given parameters --exclude-test and/or --exclude-install, if the modules are in any of them won't be tested isolately

File: .github/workflows/bt_ci_workflow_isolated_unit_tests.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            if 'exclude-install' in commands_dict:
+                commands_dict['exclude-install'] = commands_dict['exclude-install'][0].split(',')
+            if 'exclude-test' in commands_dict:
+                commands_dict['exclude-test'] = commands_dict['exclude-test'][0].split(',')
+            
+            modules_to_process = set()
+            for filename in filenames:
+                from_submodule = False            
+                extension = os.path.splitext(filename)[1]
+                if extension in ['.py', '.xml', '.js']:
```
Comment (BT-fgarbely) [inline .github/workflows/bt_ci_workflow_isolated_unit_tests.yml:53]: I would also add `.csv` files.
Because there could be errors during the installation of the module coming from `.csv` files.

I also thought about adding `.po` files, because we can have errors from there as well.
But as we don't install/use languages during tests, we can ignore `.po` files I guess.
_=> I just wanted to write down my thoughts_

`.scss`, `.css`, etc are maybe not needed as they shouldn't generate problems.
The only thing is related integration/frontend tests. I don't know if we will test the correct styles in future. Then we could need it. But I have never seen such tests in braintec (until now).

**Review by BT-fgarbely — APPROVED** [review]
Good job.
It's hard to follow the entire process without testing/debugging. But I haven't noticed any problems.

File: .github/workflows/bt_ci_workflow_isolated_unit_tests.yml (hunk 1 above)
Comment (BT-rmartin) [reply .github/workflows/bt_ci_workflow_isolated_unit_tests.yml:53]: Let's add all just in case
